packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1799,7 +1799,11 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
     _cachedController = null;
   }
 
-  void _handleTrackTapDown(TapDownDetails details) {
+  /// Handler called when the track is tapped in order to scroll to the tapped
+  /// position.
```

Comment:
I think so, right? Or does it jump to that specific position now?
```suggestion
  /// Handler called when the track is tapped in order to page in the tapped direction
  /// position.
```

Reply:
It pages, not scrolls. Thanks for the catch.